install_cpan_modules.pl

```diff
@@ -22,9 +24,39 @@
     "LWP::UserAgent",
 );
 
-# Windows-specific modules
+# Windows-specific modules and file editing
 if ( $^O eq 'MSWin32' ) {
     push @modules, "Win32::Unicode::Process";
+
+    # If your Tcl location is non-default, please adjust this path here!
+    my $filename = 'C:/Tcl/lib/tclConfig.sh';
+
+    my $data = read_file($filename);
+    my $regex = qr/([A-Z_=]+[']+[{]*[-L]*[-I]*)([\/]*[A-Z]+[:]*[\\\/]+BawtBuilds\/TclDistribution\/TclDistribution\-[\d.a]*\/Windows\/[x64|x86]+\/Release\/[Install|Build]+\/Tcl)/mp;
```

Comment:
@cpeel @windymilla There's probably a way to make this a little more elegant, but the weird placeholder shows up so many ways that I tried to catch them all with the one expression. This should also work for the 8.6 branch as well as if 8.7a if that ever starts working, and catches both x64 and x86 installations.

Reply:
In an even better world, the `$data` and `$filename` variables would be linked up so that you could just specify `C:/Tcl` or whatever once, but I couldn't quite get the regex to replace everything correctly that way. I assume the number of people who need to customize this is low enough that asking them to change two strings isn't the worst thing.

Reply:
What you have here is great and agreed that it isn't worth micro-optimizing.